README.md

```diff
@@ -328,11 +328,13 @@ Use the variables starting with the `mssql_ha_` prefix to configure an SQL Serve
 **Prerequisites**
 
 * Ensure that your hosts meet the requirements for high availability configuration, namely DNS resolution configured so that hosts can communicate using short names.
-For more information, see Prerequisites in [Configure SQL Server Always On Availability Group for high availability on Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-availability-group-configure-ha?view=sql-server-ver15#prerequisites).
-* In SQL Server, create a database to be used for replication.
-Provide the database name to the role with the [`mssql_ha_db_name`](#mssql_ha_db_name) variable.
-You can set the [`mssql_pre_input_sql_file`](#mssql_pre_input_sql_file-and-mssql_post_input_sql_file) variable to pre-create the database.
-For more information, see the description of the [`mssql_ha_db_name`](#mssql_ha_db_name) variable.
+  For more information, see Prerequisites in [Configure SQL Server Always On Availability Group for high availability on Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-availability-group-configure-ha?view=sql-server-ver15#prerequisites).
+* Optional: In SQL Server, create one or more databases to be used for replication.
+  Provide databases names to the role with the [`mssql_ha_db_names](#mssql_ha_db_names variable.
```

Comment:
formatting - I don't think you need to use backticks for strings inside the square brackets - also missing a close parenthesis

Reply:
Thank you, I need somewhat bigger glasses.
I like to use backticks there to highlight this as a code. That way, mssql_ha_db_names is formatted both as code and as an URL. This is to distinguish with other places where URL content is not a code.

Reply:
> Thank you, I need somewhat bigger glasses. I like to use backticks there to highlight this as a code. That way, mssql_ha_db_names is formatted both as code and as an URL. This is to distinguish with other places where URL content is not a code.

ok